benchmark/run_benchmarks.jl

```diff
@@ -7,7 +7,7 @@ let results = judge(Trixi,
             BenchmarkConfig(juliacmd=`$(Base.julia_cmd()) --check-bounds=no --threads=1`, id="main") # baseline
         )
 
-    export_markdown(joinpath(pathof(Trixi) |> dirname |> dirname, "benchmark", "results_threads1.md"), results)
+    export_markdown(joinpath(pathof(Trixi) |> dirname |> dirname, "benchmark", "results_$(gethostname)_threads1.md"), results)
```

Comment:
Shouldn't this be `"results_$(gethostname())_threads1.md"`? Also below...

Reply:
Yeah, I've already fixed that... I wanted to run some additional performance benchmarks for #509 and needed this also on `main` for comparison. Sorry for not waiting for your review here. I got temporary access to some nice Intel machine and wanted to check the performance there today.